pkg/kubelet/cadvisor/util_test.go

```diff
@@ -55,3 +56,45 @@ func TestCapacityFromMachineInfoWithHugePagesEnable(t *testing.T) {
 func TestCrioSocket(t *testing.T) {
 	assert.EqualValues(t, CrioSocket, crio.CrioSocket, "CrioSocket in this package must equal the one in github.com/google/cadvisor/container/crio/client.go")
 }
+
+func TestDefaultCadvisorMetricSet(t *testing.T) {
+	usingLegacyStatsTable := []bool{false, true}
```

Comment:
Thanks for creating tests!
What about true,true and false,false?

Reply:
`usingLegacyStats` just depends on kubelet runtime or remote runtime endpoint, and its value affects whether to collect disk stats in cadvisor. 
Its combination is just `true` or `false`, when set custom cadivsor metrics if `CustomCadvisorMetrics ` feature gate enabled or just like before, i.e. Default cadvisor metric set. So `true,ture` or `false,false` makes no sense.